docs/backend/guides/wip-server-side-validation.md

```diff
@@ -22,7 +22,12 @@ I propose that we begin to write these guidelines starting with validation. Once
 This is where we authenticate the user. Any code related to permissions, sessions, certificates, etc, belongs here. No validation related to user input or the request body should be in this layer.
 
 ### Swagger (OpenAPI)
-This is the first place where we'll see any input validation. After the server authenticates the user, it tries to convert our request to the Go types defined by the packages located in `./pkg/gen`. These files are generated using Swagger Codegen, a tool that analyzes our API specification and converts it into Go code. **We do not write any code in `./pkg/gen` - all of these files are autogenerated.**
+This is the first place where we'll see any input validation. After the server
+authenticates the user, it tries to convert our request to the Go types defined
+by the packages located in `./pkg/gen`. These files are generated using Swagger
+Codegen, a tool that analyzes our API specification and converts it into Go
```

Comment:
`Codegen` another term worth adding to Vale.

Reply:
yeah. #61 has a lot of these terms added, but because of the release process for Vale, my PR is stuck in limbo until a few other things are updated.